arduino/arduino-ide
# Added overwrite confirmation to ZIP lib install

This PR adds support for overwrite confirmation when installing a lib from a zip.

Steps to verify:
 - Install a lib from a zip that was not installed,
 - Install it again,
 - You should see the confirm dialog whether to overwrite or not. If you hit `No`, nothing happens. If you hit `Yes`, the existing lib will be overwritten.

Please verify. Thank you!

Signed-off-by: Akos Kitta <kittaakos@typefox.io>
